File: src/main/java/com/pickme/calendar/CalendarApplication.java
package com.pickme.calendar;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@EnableMongoRepositories // mongodb 사용하기 위함
@SpringBootApplication
public class CalendarApplication {

	public static void main(String[] args) {
		SpringApplication.run(CalendarApplication.class, args);
	}

}

File: src/main/kotlin/com/pickme/calendar/CalendarApplication.kt
package com.pickme.calendar

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories

@EnableMongoRepositories // mongodb 사용하기 위함
@SpringBootApplication
class CalendarApplication

fun main(args: Array<String>) {
    runApplication<CalendarApplication>(*args)
}